packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, nodeindex, pkg_name="", pkg_version="", namespace = ""):
         self.add_token(Token("", TokenKind.SkipDiffRangeStart))
         self.add_literal(HEADER_TEXT)
         self.add_token(Token("", TokenKind.SkipDiffRangeEnd))
-        self.add_new_line(2)
+        self.add_newline()
```

Comment:
Do we need this since you are adding 2 new lines below and clearly previous token was not new line.

Reply:
Yes, we need the newline after the header text.

Reply:
Fixed.